src/js/utils/option-utils.jsx

```diff
@@ -114,8 +114,8 @@ export const debounceAvailableItemsFetch = (waitTime, minSearchLength) =>
             quantityAvailable: obj.quantityAvailable,
             minExpirationDate: obj.minExpirationDate,
             handlingIcons: obj.product.handlingIcons,
-            label: `${obj.productCode} - ${obj.translatedName ?? obj.name}`,
-            hasTranslatedName: !!obj.translatedName,
+            label: `${obj.productCode} - ${obj?.product.displayNames?.default ?? obj.name}`,
+            hasTranslatedName: !!obj?.product?.displayNames?.default,
```

Comment:
shouldn't it be renamed to:
```suggestion
            hasDisplayName: !!obj?.product?.displayNames?.default,
```

or even as we've discussed in my pr, just keep:
```javascript
displayName: obj?.product?.displayNames?.default,
```